openedx/core/djangoapps/user_api/accounts/settings_views.py

```diff
@@ -119,7 +119,10 @@ def account_settings_context(request):
         'show_program_listing': ProgramsApiConfig.is_enabled(),
         'show_dashboard_tabs': True,
         'order_history': user_orders,
-        'disable_order_history_tab': REDIRECT_TO_ORDER_HISTORY_MICROFRONTEND.is_enabled(),
+        'disable_order_history_tab': (
+            REDIRECT_TO_ORDER_HISTORY_MICROFRONTEND.is_enabled() and
+            configuration_helpers.get_value('ENABLE_ORDER_HISTORY_MICROFRONTEND')
```

Comment:
To DRY this, can we add a method in `openedx.core.djangoapps.user_api.accounts.toggles` that encapsulates checking for both site-configuration and django settings so it's clear in the `toggles` module that we're checking both?  For example, `should_redirect_to_order_history_microfrontend()`?

Reply:
Sure.